liberty-maven-plugin/src/it/setup/pom.xml

```diff
@@ -12,7 +12,23 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
     </properties>
-
+    
+    <build>
+       <pluginManagement>
```

Comment:
How about just add the compile level properties to the parent pom and we don't need to update the compile plugin version in the future. 
```xml
        <maven.compiler.source>1.7</maven.compiler.source>
        <maven.compiler.target>1.7</maven.compiler.target>
```